scalatest/src/main/scala/org/scalatest/PrivateMethodTester.scala

```diff
@@ -240,7 +240,7 @@ trait PrivateMethodTester {
       // Store in an array, because may have both isEmpty and empty, in which case I
       // will throw an exception.
       val methodArray =
-        for (m <- target.getClass.getDeclaredMethods; if isMethodToInvoke(m))
+        for (m <- target.getClass.getDeclaredMethods.map(_.nn); if isMethodToInvoke(m))
```

Comment:
**Java method returns array**
Java method returns array of type `Array[Null | T]`. We need to map it to `Array[T]`.